src/gui/C4StartupScenSelDlg.cpp

```diff
@@ -1499,9 +1514,7 @@ C4StartupScenSelDlg::C4StartupScenSelDlg(bool fNetwork) : C4StartupDlg(LoadResSt
 	SetFocus(pScenSelList, false);
 
 	// key bindings
-	C4CustomKey::CodeList keys;
-	keys.emplace_back(K_BACK); keys.emplace_back(K_LEFT);
-	pKeyBack = new C4KeyBinding(keys, "StartupScenSelFolderUp", KEYSCOPE_Gui,
+	pKeyBack = new C4KeyBinding(C4KeyCodeEx(K_LEFT), "StartupScenSelFolderUp", KEYSCOPE_Gui,
```

Comment:
I do not have experience with the GUI stuff, could you explain what you did there? Apparently now the K_BACK is not used anymore - what effect does that have?
From my naive point of view the change does not look equivalent to the previous version.

Reply:
K_BACK is the backspace key which could be used to switch back the same way you could (and still can) do it with the left arrow key - however, it also switched back when the search bar had the focus, making text deletion impossible.

Reply:
Ok, makes sense